src/sst/core/elemLoader.h

```diff
@@ -40,9 +35,17 @@ class ElemLoader
     void loadLibrary(const std::string& elemlib, std::ostream& err_os);
 
     /**
-     * Returns a list of potential element libraries in the search path
+     * Search paths for potential elements and add them to the provided vector
+     *
+     * @param potElems - vector of potential elements that could contain elements
+     * @return void
      */
-    std::vector<std::string> getPotentialElements();
+    void getPotentialElements(std::vector<std::string>& potElems);
```

Comment:
If this is going to be used only by sstinfo. why not just have it return a std::vector instead of pass by reference for output? As in what don't you like about the old behavior?

Reply:
We discsussed this on Mattermost. If we proceed to the App-Store/Model-Store approach, we will have a `std::vector` and just call methods to treat them as an append-like operation.

Reply:
Is this PR the right place to do this?